Adds bearer token support for mimirtool's analyze ruler/prometheus co… (#9587)
grafana/mimir: docs/sources/mimir/manage/tools/mimirtool.md

##### Configuration

| Environment variable | Flag            | Description                                                                                         |
| -------------------- | --------------- | --------------------------------------------------------------------------------------------------- |
| `MIMIR_ADDRESS`      | `--address`     | Sets the address of the Prometheus instance.                                                        |

> | `MIMIR_TENANT_ID`    | `--id`          | Sets the basic auth username. If you're using Grafana Cloud, this variable is your instance ID.     |

Reviewer: ```suggestion
| `MIMIR_TENANT_ID`    | `--id`          | Sets the basic authentication username. If you're using Grafana Cloud, this variable is your instance ID.     |
```

> | `MIMIR_API_KEY`      | `--key`         | Sets the basic auth password. If you're using Grafana Cloud, this variable is your API key.         |

Reviewer: ```suggestion
| `MIMIR_API_KEY`      | `--key`         | Sets the basic authentication password. If you're using Grafana Cloud, this variable is your API key.         |
```

> | `MIMIR_AUTH_TOKEN`   | `--auth-token`  | Sets bearer or JWT token that is required for Mimir clusters authenticating by bearer or JWT token. |

Reviewer: ```suggestion
| `MIMIR_AUTH_TOKEN`   | `--auth-token`  | Sets the bearer or JWT token that is required for Mimir clusters authenticating with this method. |
```
